pkg/validating-webhooks/hive/v1/clusterdeployment_validating_admission_hook.go

```diff
@@ -26,6 +26,7 @@ import (
 
 	hivev1 "github.com/openshift/hive/apis/hive/v1"
 	hivev1aws "github.com/openshift/hive/apis/hive/v1/aws"
+	"github.com/openshift/hive/apis/hive/v1/aws"
```

Comment:
`make update-gofmt` wants to sort this for `make verify`

Reply:
Sigh.

For the sake of expediency, Imma let e2e* finish before I push this change; then if `verify` passes I'll mash the merge button, wfy?